react-native-cli/index.js

```diff
@@ -140,7 +144,15 @@ function createProject(name) {
       process.exit(1);
     }
 
-    var cli = require(CLI_MODULE_PATH());
+    cli = require(CLI_MODULE_PATH());
     cli.init(root, projectName);
   });
 }
+
+function checkForVersionArgument() {
+  if (process.argv.indexOf('-v') >= 0 || process.argv.indexOf('--version') >= 0) {
+    var pjson = require('./package.json');
+    console.log(pjson.version);
+    process.exit(1);
```

Comment:
I believe this should be `process.exit()` - other commands above that exit with status code 1 are because they are errors :smile:

See: https://nodejs.org/api/process.html#process_process_exit_code